api/src/wfl/service/rawls.clj

```diff
@@ -136,19 +136,13 @@
                                :attributeListName list-name}
                   make-member (partial assoc template :newMember)]
               (reduce #(conj %1 (make-member %2)) [init] v)))
-          (no-op [] [])
-          (on-unhandled-attribute [name value]
-            (throw (ex-info "No method to make upsert operation for attribute"
-                            {:name name :value value})))
           (to-operations [attributes]
             (flatten
              (for [[k v] (seq attributes)]
-               (cond (number? v) (add-scalar k v)
-                     (string? v) (add-scalar k v)
-                     (map?    v) (add-scalar k v)
-                     (coll?   v) (add-list   k v)
-                     (nil?    v) (no-op)
-                     :else       (on-unhandled-attribute k v)))))
+               (cond (map?  v) (add-scalar k v)
```

Comment:
(Not important but curious) is there a reason that this conditional is broken out separately, rather than falling back to the `:else`?

Reply:
A map is a collection but we handle it as an "object".
COND processes clauses in order so the map? test has to come before coll?.
(Yes. I had to debug that mistake so it is fresh in mind.)